plugins/woocommerce/tests/api-core-tests/tests/customers/customers-crud.test.js

```diff
@@ -0,0 +1,451 @@
+		});
+	});
+
+	test.describe('Retrieve after create', () => {
+		test('can retrieve a customer', async ({
+			request
+		}) => {
+			// call API to retrieve the previously saved customer
+			const response = await request.get(
+				`/wp-json/wc/v3/customers/${customerId}`
+			);
+			const responseJSON = await response.json();
+			expect(response.status()).toEqual(200);
+			expect(responseJSON.id).toEqual(customerId);
+			expect(responseJSON.is_paying_customer).toEqual(false);
+			expect(responseJSON.role).toEqual('customer');
+		});
+
+		test('can retrieve all customers', async ({
+			request
+		}) => {
+			// call API to retrieve all customers
+			const response = await request.get('/wp-json/wc/v3/customers');
+			const responseJSON = await response.json();
+			expect(response.status()).toEqual(200);
+			expect(Array.isArray(responseJSON));
+			expect(responseJSON.length).toBeGreaterThan(0);
+		});
+	});
+
+	test.describe('Update a customer', () => {
+
+		test(`can update the admin user/customer`, async ({
+			request,
+		}) => {
+			// update customer names (regular, billing and shipping) to admin
+			// (these were initialised blank)
+			const response = await request.put(
+				`/wp-json/wc/v3/customers/1`, {
+					data: {
+						first_name: 'admin',
+						billing: {
+							first_name: 'admin'
+						},
+						shipping: {
+							first_name: 'admin'
+						}
+					}
+				}
+			);
+			const responseJSON = await response.json();
+			expect(response.status()).toEqual(200);
+			expect(responseJSON.first_name).toEqual('admin');
+			expect(responseJSON.billing.first_name).toEqual('admin');
+			expect(responseJSON.shipping.first_name).toEqual('admin');
+		});
+
+		test('retrieve after update admin', async ({
+			request
+		}) => {
+			// call API to retrieve all customers
+			const response = await request.get('/wp-json/wc/v3/customers/1');
+			const responseJSON = await response.json();
+			expect(response.status()).toEqual(200);
+			expect(responseJSON.first_name).toEqual('admin');
+			expect(responseJSON.billing.first_name).toEqual('admin');
+			expect(responseJSON.shipping.first_name).toEqual('admin');
+		});
+
+		test(`can update the subscriber user/customer`, async ({
+			request,
+		}) => {
+			// update customer names (billing and shipping) to Jane
+			// (these were initialised blank, only regular first_name was populated)
+			const response = await request.put(
+				`/wp-json/wc/v3/customers/2`, {
+					data: {
+						billing: {
+							first_name: 'Jane'
+						},
+						shipping: {
+							first_name: 'Jane'
+						}
+					}
+				}
+			);
+			const responseJSON = await response.json();
+			expect(response.status()).toEqual(200);
+			expect(responseJSON.first_name).toEqual('Jane');
+			expect(responseJSON.billing.first_name).toEqual('Jane');
+			expect(responseJSON.shipping.first_name).toEqual('Jane');
+		});
+
+		test('retrieve after update subscriber', async ({
+			request
+		}) => {
+			// call API to retrieve all customers
```

Comment:
Minor update to the comment here:

```suggestion
			// call API to retrieve the subscriber customer we updated above
```